extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -125,7 +125,6 @@ tags: {  # <amp-video> not in amp-story.
     supported_layouts: FIXED
     supported_layouts: FIXED_HEIGHT
     supported_layouts: FLEX_ITEM
-    supported_layouts: INTRINSIC
```

Comment:
@honeybadgerdontcare Was this inadvertently removed? I added it here in https://github.com/ampproject/amphtml/pull/28349.

Reply:
@westonruter yes that appears so. Adding back in #31800